compiler/Eta/Main/TidyPgm.hs

```diff
@@ -617,12 +617,12 @@ chooseExternalIds hsc_env mod omit_prags expose_all binds implicit_binds imp_id_
  where
   nc_var = hsc_NC hsc_env
 
-  -- init_ext_ids is the intial list of Ids that should be
+  -- init_ext_ids is the initial list of Ids that should be
   -- externalised.  It serves as the starting point for finding a
   -- deterministic, tidy, renaming for all external Ids in this
   -- module.
   --
-  -- It is sorted, so that it has adeterministic order (i.e. it's the
+  -- It is sorted, so that it has a deterministic order (i.e. it's the
```

Comment:
?

`adeterministic` would mean `without deterministic`, which didn't seem right.

In general in this repository, I stayed far away from single character prefixed tokens as they appeared to be intentional. This was an exception.